app/presenters/signup_presenter.rb

```diff
@@ -35,7 +35,7 @@ def email_filter_by
   end
 
   def can_modify_choices?
-    choices? && choices_formatted.any?
+    choices? && choices_formatted.any? && !content_item["details"]["cannot_modify_choices"]
```

Comment:
There are other finders which don't present a list of facets to change, eg https://www.gov.uk/search/news-and-communications/email-signup?level_one_taxon=e491505c-77ae-45b2-84be-8c94b94f6a2b&organisations%5B%5D=academy-for-social-justice&order=updated-newest, how do they do it?  Can we reuse the same mechanism?  Is that impossible because specialist finders do email in a different way?

I'm really not a fan of a negatively-named boolean, but I see your point about not wanting to change all the existing content items.  If this is necessary for the specialist finders, then I think it should be documented in the card about making email signups consistent as something to remove when that work is done.

Maybe it's even worth having a card about making specialist and non-specialist finder email signups consistent, and leaving the wider email signup consistency problems for after this smaller problem is solved.

Reply:
@barrucadu Maxwell did indeed mention that some followup work will have to be done to make the email alert signups consistent.

As far as I can see, the divide isn't so much between finders and specialist finders, but between finders that use so-called dynamic filters (like News and Communications), and the rest: https://github.com/alphagov/finder-frontend/blob/master/lib/email_alert_title_builder.rb#L123.

Reply:
I don't mind doing something tactical here.  We're flagging that there's a general lack of consistency (and that the signup journey needs work) across email journeys.  So long as we can keep a note of what we've done in commit messages and some product docs, that's probs OK.

Reply:
(Linting's failing by the way)

Reply:
Thanks @sihugh, I'll document that.